docs/ANONYMITY_NETWORKS.md

```diff
@@ -10,7 +10,13 @@ relying on IPv4 for the remainder of messages to make surrounding node attacks
 
 ## Behavior
 
-If _any_ anonymity network is enabled, transactions being broadcast that lack
+When the `--proxy` flag is used, `monerod` sends all node traffic via the proxy.
```

Comment:
It should send all external connections with this flag, which includes checking for latest version, etc.